certdehydrate/certdehydrate.go

```diff
@@ -0,0 +1,231 @@
+package certdehydrate
+
+import "bytes"
+import "crypto/sha256"
+import "crypto/x509/pkix"
+import "encoding/base64"
+import "encoding/binary"
+import "fmt"
+import "math/big"
+import "time"
```

Comment:
for future reference, I wrote imports in this style before learning the preferred style:

```
import (
  "..."
  "..."
)
```

I switch to this new style as I encounter code with it.